src/compas/geometry/transformations/matrices.py

```diff
@@ -397,6 +399,55 @@ def matrix_from_frame(frame):
     return M
 
 
+def matrix_from_frame_to_frame(frame_from, frame_to):
+    """Computes a transformation between two frames.
+
+    This transformation allows to transform geometry from one Cartesian
+    coordinate system defined by "frame_from" to another Cartesian
+    coordinate system defined by "frame_to".
+
+    Parameters
+    ----------
+    frame_from : :class:`Frame`
+        A frame defining the original Cartesian coordinate system
+    frame_to : :class:`Frame`
+        A frame defining the targeted Cartesian coordinate system
+
+    Examples
+    --------
+    >>> f1 = Frame([2, 2, 2], [0.12, 0.58, 0.81], [-0.80, 0.53, -0.26])
+    >>> f2 = Frame([1, 1, 1], [0.68, 0.68, 0.27], [-0.67, 0.73, -0.15])
+    >>> T = matrix_from_frame_to_frame(f1, f2)
+    """
+    T1 = matrix_from_frame(frame_from)
+    T2 = matrix_from_frame(frame_to)
+    return multiply_matrices(T2, matrix_inverse(T1))
+
+
+def matrix_change_basis(frame_from, frame_to):
```

Comment:
how is this different than `frame_to_frame`?

Reply:
the `frame_to_frame` transformation from frame f1 to frame f2 allows to transform geometry from f1 into f2, i.e. the orientation of the object in regards to the respective frame stays the same.
The `change_basis` transformation from frame f1 to frame f2 allows to represent geometry of frame f1 in frame f2 (not transforming the geometry )
so they are used for different purposes...